assets/quick-add.css

```diff
@@ -38,6 +38,10 @@
   width: 90%;
 }
 
+.quick-add-modal .variant-item__image-container.global-media-settings::after {
```

Comment:
Hmmm. I don't understand how the QOL images would have shadows on main, but not my branch. My code change specifically only targets the quick add modal, and I see the `::after` element is still trying to apply a shadow on the QOL images:

![image](https://github.com/Shopify/dawn/assets/590055/7dfda771-4521-4ae5-a239-b42b7b8d4209)

Reply:
Sorry what I meant is that the QOL doesnt have shadows on main **and** your branch which is correct :) (as per Meli's ticket - cc @melissaperreault if I misunderstood this)

But since they already dont have the shadows on main for QOL and this is for QOL bulk buy (which uses the same snippet), i dont think we need to override it specifically like you have on your branch

I think I found the culprit: https://screenshot.click/01-44-1difj-133ys.mp4

What im saying is we should do this instead to target both:

```
.variant-item__image-container.global-media-settings::after {
  content: none;
}
```

Because we dont want shadows in quick add bulk **and** qol

Reply:
Ah, yes, good catch. And I can move the fix to the quick-order-list CSS since it's used in both places 🎉